ui/pages/import-token/index.scss

```diff
@@ -59,4 +59,9 @@
     margin-bottom: 16px;
     margin-top: 0;
   }
+
+  &__collectible-address-error-link {
+    color: var(--primary-blue);
+    cursor: pointer;
+  }
```

Comment:
Should be blue instead of orange on hover.

<img width="1440" alt="Screen Shot 2022-01-17 at 10 17 15 AM" src="https://user-images.githubusercontent.com/8112138/149826127-be05b7f4-36af-425a-aac0-0f465b739425.png">

Might have to add this extra css to get the link to hover correctly. I've created an issue for this #13335

```suggestion
  &__collectible-address-error-link {
    color: var(--primary-blue);
    cursor: pointer;

    &:hover {
      color: var(--Blue-400);
    }
  }
```